packages/dui3/package.json

```diff
@@ -26,20 +26,27 @@
     "@apollo/client": "^3.7.14",
     "@headlessui/vue": "^1.7.13",
     "@heroicons/vue": "^2.0.12",
+    "@jsonforms/core": "3.1.0",
+    "@jsonforms/vue": "3.1.0",
+    "@jsonforms/vue-vanilla": "3.1.0",
     "@pinia/nuxt": "^0.4.11",
     "@speckle/shared": "workspace:^",
     "@speckle/ui-components": "workspace:^",
     "@speckle/ui-components-nuxt": "workspace:^",
     "@vue/apollo-composable": "^4.0.0-beta.5",
     "@vueuse/core": "^9.13.0",
     "apollo-upload-client": "^17.0.0",
+    "crypto-random-string": "^5.0.0",
```

Comment:
crypto-random-string relies on node/backend dependencies, do not use it on the client-side. If you need any random ID, `nanoid` is a good & lightweight alternative